docs/_sources/compatibility.rst.txt

```diff
@@ -0,0 +1,21 @@
+Plugin Compatibility
```

Comment:
Maybe add a piece here that if their plugin doesn't work to make an issue and maybe make an issue tag called "plugin compatibility"